decidim-proposals/app/views/decidim/proposals/proposals/_tags.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if proposal.category.present? || proposal.scope.present? %>
```

Comment:
@oriolgual didn't you do it yourself? I see this already done here at GitHub, but wasn't done on my local machine 😕